RTNeural/gru/gru.tpp

```diff
@@ -3,6 +3,8 @@
 namespace RTNeural
 {
 
+constexpr int kNumBiasLayers = 2;
```

Comment:
Since the `.tpp` file is included in a header file [elsewhere](https://github.com/jatinchowdhury18/RTNeural/blob/2fd2c309c47aba6290b95a025a8d833cca81bc18/RTNeural/Model.h#L13), defining the constant here will define it throughout the `RTNeural` namespace. I think the correct scope for this constant would be to make it a member variable of the `GRULayer` class. IIRC, `constexpr` member variables need to also be marked `static`.

Reply:
Will do!